package.json

```diff
@@ -20,7 +20,9 @@
     "test:e2e": "jest --config ./test/jest-e2e.json",
     "migration:generate": "ts-node node_modules/.bin/typeorm migration:generate",
     "migration:run": "ts-node node_modules/.bin/typeorm migration:run",
-    "migration:revert": "ts-node node_modules/.bin/typeorm migration:revert"
+    "migration:revert": "ts-node node_modules/.bin/typeorm migration:revert",
+    "doc:serve": "npx compodoc -p tsconfig.json -w -s -r 8001",
+    "doc:generate": "npx compodoc -p tsconfig.json"
   },
   "dependencies": {
```

Comment:
@asad-mlbd Should we set the `dependencies` to fixed versions?
Especially for `swagger`, else it might break again.
`"@nestjs/swagger": "4.5.1",`